### ISSUE TITLE
Doesn't seem to work

### ISSUE THREAD

**Issue by jathek** [Nonary/RTSSLimiter#1]
Using the v1.0.0 release, but RTSS doesn't seem to be setting a limit. The ResolutionAutomation script from the other repo is working, so I don't think it's an install issue?

If I'm interpreting the Moonlight stats correctly, it's requesting a 60fps stream, but the RTSS overlay shows 98fps in the game.

![image](https://github.com/user-attachments/assets/e815cdb6-86d0-435b-8fdf-5a4f9b5efc9e)
![image](https://github.com/user-attachments/assets/3ea250d1-213e-4d9f-b66a-66b63d33e911)
![image](https://github.com/user-attachments/assets/67ac876d-01a9-4643-a3b3-d4a64324369b)

**Comment by Nonary** [comment]
I am not sure how RTSSLimiter got installed there with config.elevated unchecked. It requires elevation in order to work

**Comment by jathek** [comment]
I deleted the rule in Sunshine and reinstalled, and now config.elevated is checked. It's still not limiting, though.

**Comment by Nonary** [comment]
> I deleted the rule in Sunshine and reinstalled, and now config.elevated is checked. It's still not limiting, though.

make sure you dont have another profile configured for the game on RTSS and that you have not turned off the frame limit option

**Comment by jathek** [comment]
![image](https://github.com/user-attachments/assets/aafb15c9-dda3-48a1-8a55-d4825ceff8d1)
No other profiles, and I've tried with multiple games. With these settings, when I start a Moonlight 30fps stream, the framerate stays at 60fps. I think it _should_ be decreasing to 30fps?

EDIT: It finally started working. It did reduce the fps to 30 on my latest test. I previously wasn't using ResolutionAutomation, and I reinstalled it. Maybe that was the difference?
